nexus/social_media/services.py

```diff
@@ -30,5 +77,30 @@ def post_to_facebook(post_id):
             page_graph.put_photo(image=post.image.file.open('rb'))
     elif post.text:
         page_graph.put_object(
-            parent_object=settings.FB_PAGE_ID, connection_name='feed',  message=post.text
+            parent_object=settings.FB_PAGE_ID, connection_name='feed', message=post.text
+        )
+
+
+def publish_on_social_media():
+    if settings.LIMIT_POSTS is True and int(settings.MAX_POSTS_AT_ONCE) > 0:
+        posts = Post.objects.filter(
```

Comment:
Why not just save the queryset in a variable and slice & evaluate it only when the above conditions are `True`?

Let's use KISS and DRY principles in mind :)